index.js

```diff
@@ -1,9 +1,28 @@
+const TAGS = [
+  {
+    tagRegex: /data-mc-repeatable/g,
+    replaceText: 'mc:repeatable'
+  },
+  {
+    tagRegex: /data-mc-edit/g,
+    replaceText: 'mc:edit'
+  }
+];
+
+var replaceTag = function (str, tagRegex, replaceText) {
+  return str.replace(tagRegex, replaceText);
+};
+
 exports = module.exports = function (input) {
   input = input || '';
 
   if (typeof (input) !== 'string') {
     throw new Error('invalid input for mailchimpify');
   }
 
-  return input.replace(/data-mc-edit/g, 'mc:edit');
+  TAGS.forEach(function (tag) {
+    input = replaceTag(input, tag.tagRegex, tag.replaceText);
```

Comment:
If the `tag` object gets bigger / more keys, it would be make this really long. Just pass in the `tag` object and handle it in `replaceTag`.